### DISCUSSION TITLE
MFA and QR Codes

### DISCUSSION THREAD

**Discussion by Rod-in-NM** [community/community#54818] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I've got to start using an authenticator app with my GitHub account. However, I've got two GitHub accounts and at this point only one authenticator app, which I want to use with my other GitHub account. So, I'll choose another authenticator app. When signing up to use 2FA with this GitHub account, will it generate a QR Code that I can scan into an authenticator app in order to help configure my account in the authenticator app?

**Comment by iamreiyn** [comment]
Hi @Rod-in-NM

Yes, you can use the same authentication app for multiple GitHub accounts. The authentication app generates time-based one-time passwords (TOTPs) that are associated with a particular account. As long as the TOTP is valid and matches the code requested by GitHub, you can access that specific account.

To set up, follow these steps:

1. Sign in to your GitHub account and go to your account settings.
2. Click on the "Password and authenticator" tab under "Access"
3. Next to "Two-factor authentication," click "Enable"
4. Choose the "Authentication App" option and follow the instructions to set up your authentication app.
5. When prompted, scan the QR code with your authentication app and fill in the TOTP to complete the process.

You can repeat this process for multiple GitHub accounts, using the same authentication app. Make sure to keep track of which TOTP code is associated with each account by labeling them.

If I was able to help you, kindly consider "Marking this as an answer" so people with the same question can find the appropriate answer. If you need additional help, feel free to let me know! Thank you.

**Reply by rfalanga** [reply]
@montymahato I'm trying to do this, but when I log into my GitHub account, then click on Settings I don't see a Security tab. From the Settings page I don't see a "Security" tab. I do see a "Security | Code security and analysis", which I'm sure isn't the correct place.

I did see a link on the left side for Account, so I clicked on that hoping it would reveal a Security tab. It isn't on the menu on the left side of the page. And on the main body of the page, it has four sections: Change username, Export account data, Successor settings and Delete account.

I also tried searching for "two-factor", but only got a list of several repos - which didn't help. Where is the Security page that you referred to, please?

**Reply by airtower-luna** [reply]
The menu entry in user settings is actually called "Password and authentication". Not sure if that changed recently.

**Reply by Rod-in-NM** [reply]
I found it. "Password and authentication" are under Access.

**Reply by iamreiyn** [reply]
My bad, I've updated the answer.. forgot while writing. @airtower-luna @rfalanga
